compiler/dyno/util/linters/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+# Copyright 2021-2022 Hewlett Packard Enterprise Development LP
+# Other additional copyright holders may be indicated within.
+#
+# The entirety of this work is licensed under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+#
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+add_executable(fieldsUsed "fieldsUsed.cpp")
+
+execute_process(COMMAND llvm-config --cxxflags OUTPUT_VARIABLE llvm_cxx_flags OUTPUT_STRIP_TRAILING_WHITESPACE)
```

Comment:
Yeah the first call to llvm-config isn't used and I've removed it. I pushed a commit to use the find_packages cmake builtin instead of printchplenv b/c I don't think this needs to be tied to the LLVM we use elsewhere and a bit of a testing ground for what a post printchplenv world might look like

Reply:
(an aside, but even if we use cmake everywhere, we might still have printchplenv - for one thing it indicates how `chpl` should behave when you run it)

Reply:
Anyway, I'm OK with using a cmake find-llvm thing here for now.